soh/soh/Enhancements/randomizer/location_list.cpp

```diff
@@ -1594,10 +1594,10 @@ void Rando::StaticData::InitLocationTable() { //
 	locationTable[RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_1] 	    	= Location::Pot(RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_1, 		        RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		    SCENE_WATER_TEMPLE, 		    TWO_ACTOR_PARAMS(28, -3817), 	"MQ Mini Dodongo Pot 1", 		"Water Temple MQ Mini Dodongo Pot 1", 	    	    RHT_POT_WATER_TEMPLE,	            RG_BOMBS_5,         SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_MINI_DODONGO_POT_1));
 	locationTable[RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_2] 	    	= Location::Pot(RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_2, 		        RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		    SCENE_WATER_TEMPLE, 		    TWO_ACTOR_PARAMS(-28, -3817), 	"MQ Mini Dodongo Pot 2", 		"Water Temple MQ Mini Dodongo Pot 2", 	    	    RHT_POT_WATER_TEMPLE,	            RG_BOMBS_5,         SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_MINI_DODONGO_POT_2));
-	locationTable[RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_1]   = Location::Pot(RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_1,     RCQUEST_MQ, RCAREA_GERUDO_TRAINING_GROUND,	SCENE_GERUDO_TRAINING_GROUND,   TWO_ACTOR_PARAMS(-324, -72),    "MQ Lobby Left Pot 1",          "Gerudo Training Grounds MQ Lobby Left Pot 1", 	    RHT_POT_GERUDO_TRAINING_GROUNDS,    RG_MAGIC_SINGLE,    SpoilerCollectionCheck::RandomizerInf(RAND_INF_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_1));
+	locationTable[RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_1]   = Location::Pot(RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_1,     RCQUEST_MQ, RCAREA_GERUDO_TRAINING_GROUND,	SCENE_GERUDO_TRAINING_GROUND,   TWO_ACTOR_PARAMS(-324, -72),    "MQ Lobby Left Pot 1",          "Gerudo Training Grounds MQ Lobby Left Pot 1", 	    RHT_POT_GERUDO_TRAINING_GROUNDS,    RG_BLUE_RUPEE,      SpoilerCollectionCheck::RandomizerInf(RAND_INF_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_1));
     locationTable[RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_2]   = Location::Pot(RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_2,     RCQUEST_MQ, RCAREA_GERUDO_TRAINING_GROUND,	SCENE_GERUDO_TRAINING_GROUND,   TWO_ACTOR_PARAMS(-324, -177),   "MQ Lobby Left Pot 2",          "Gerudo Training Grounds MQ Lobby Left Pot 2", 	    RHT_POT_GERUDO_TRAINING_GROUNDS,    RG_GREEN_RUPEE,     SpoilerCollectionCheck::RandomizerInf(RAND_INF_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_2));
     locationTable[RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_RIGHT_POT_1]  = Location::Pot(RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_RIGHT_POT_1,    RCQUEST_MQ, RCAREA_GERUDO_TRAINING_GROUND,	SCENE_GERUDO_TRAINING_GROUND,   TWO_ACTOR_PARAMS(199, -79),     "MQ Lobby Right Pot 1",         "Gerudo Training Grounds MQ Lobby Right Pot 1",     RHT_POT_GERUDO_TRAINING_GROUNDS,    RG_GREEN_RUPEE,     SpoilerCollectionCheck::RandomizerInf(RAND_INF_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_RIGHT_POT_1));
```

Comment:
While you are making changes in here, these two other pots are incorrectly listed as green rupees. They are actually recovery hearts, but the game will switch the heart to a green rupee if you have full health.

```suggestion
    locationTable[RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_2]   = Location::Pot(RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_2,     RCQUEST_MQ, RCAREA_GERUDO_TRAINING_GROUND,	SCENE_GERUDO_TRAINING_GROUND,   TWO_ACTOR_PARAMS(-324, -177),   "MQ Lobby Left Pot 2",          "Gerudo Training Grounds MQ Lobby Left Pot 2", 	    RHT_POT_GERUDO_TRAINING_GROUNDS,    RG_RECOVERY_HEART,     SpoilerCollectionCheck::RandomizerInf(RAND_INF_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_LEFT_POT_2));
    locationTable[RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_RIGHT_POT_1]  = Location::Pot(RC_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_RIGHT_POT_1,    RCQUEST_MQ, RCAREA_GERUDO_TRAINING_GROUND,	SCENE_GERUDO_TRAINING_GROUND,   TWO_ACTOR_PARAMS(199, -79),     "MQ Lobby Right Pot 1",         "Gerudo Training Grounds MQ Lobby Right Pot 1",     RHT_POT_GERUDO_TRAINING_GROUNDS,    RG_RECOVERY_HEART,     SpoilerCollectionCheck::RandomizerInf(RAND_INF_GERUDO_TRAINING_GROUNDS_MQ_LOBBY_RIGHT_POT_1));
```

Reply:
Cal had a sheet with all the parameters with a way to translate them into an item. I'm sure there's others that aren't entirely accurate but the sometimes random, sometimes conditional nature meant having to make some decisions. This green rupee/heart is a perfect example, either could work IMO even after your explanation. The magic refill was skipped at the time because we didn't really feel like it was worth the effort adding a custom randomizer item for it but that's obviously subjective.

That said I really don't feel these are very impactful anyway, so in the end I'm happy with whatever we decide to go with in the end.